src/middlewares/growthbook.ts

```diff
@@ -0,0 +1,17 @@
+import {Context} from 'koa';
+import {getGrowthbook, getRequestContext} from '../index';
+import {getLogger} from '../initializers/log4js';
+
+const logger = getLogger('orka.middlewares.growthbook');
+
+export default async function growthbook(ctx: Context, next: () => Promise<void>) {
+  if (ctx.path === '/health') return next();
+  const gb = getGrowthbook();
+  if (!gb) return next();
+  ctx.state.growthbook = gb;
+  await gb.loadFeatures().catch(e => logger.error(e, 'failed to load growthbook'));
```

Comment:
🔧 This adds a delay to each and every http call. 
Removed await in job-board for this reason, features might be delayed a few ms the first time a service starts up

Reply:
I know. I saw the comments, but I don't think it is orka's decision to not await this call. Some apps may need to have features loaded from the first call. 

In any case, this caches the requests to growthbook correct?

Reply:
Not sure if it was only the first call however or all of them. Do you mind checking?

Reply:
I'll check